src/components/atoms/DonutChart.svelte

```diff
@@ -1,47 +1,77 @@
 <script>
     // Props
     export let innerRadius
-    // export let data
 
-    /**
-     * todo
-     * give height/width with props
-     * calculate transform/outer radius based on width and height
-     * give colour array with props?
-     */
+    // Internals
+    import { reportData } from '@/stores/dataStore.js'
 
-    let data = [
-            {
-                'size': 25,
-                'name': 'Agressie',
-                'color': '#05006E'
-            },
-            {
-                'size': 8,
-                'name': 'Aanranding',
-                'color': '#492B9E'
-            },
-            {
-                'size': 12,
-                'name': 'Anders',
-                'color': '#7D43F9'
-            },
-            {
-                'size': 2,
-                'name': 'Anders',
-                'color': '#8760DB'
-            },
-            {
-                'size': 39,
-                'name': 'Anders',
-                'color': '#AA83FF'
-            },
-            {
-                'size': 11,
-                'name': 'Anders',
-                'color': '#D6C8F4'
+    // Select only the accidents from each report
+    let rawData
+    reportData.subscribe((value) => {
+        rawData = value
+    })
+
+    // Map over all the reports and return them as arrays with objects
+    const allReports = rawData.map((item) => item.accident)
+
+     // Create empty objects which will be used to count the
+     // number of reports and to build the donut chart
+    const getObject = (DynamischeNaam) => {
```

Comment:
Can you make the parameter in English instead of Dutch? Then we keep the entire code in English 😄